android/beagle/src/main/java/br/com/zup/beagle/android/components/Image.kt

```diff
@@ -99,19 +95,19 @@ data class Image(
     }
 
     @SuppressLint("CheckResult")
-    private fun getGlideRequestOptions(): RequestOptions {
+    private fun getGlideRequestOptions(placeholder: String?): RequestOptions {
         val requestOptions = RequestOptions()
         getPlaceholder(placeholder)?.let {
             requestOptions.placeholder(it)
         }
         return requestOptions
     }
 
-    private fun getPlaceholder(image: PathType.Local?): Int? {
+    private fun getPlaceholder(placeholder: String?): Int? {
```

Comment:
Sugestão para reduzir o código e aproveitar mais o Kotlin. 

```suggestion
    private fun getPlaceholder(placeholder: String?): Int? =
        placeholder?.let{
            BeagleEnvironment.beagleSdk.designSystem?.image(it)
        }
```